Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveToMinHeightCommand.java

```diff
@@ -28,14 +28,17 @@ public MoveToMinHeightCommand(ElevatorSubsystem elevator, PIDFactory pf) {
     @Override
     public void initialize() {
         log.info("Initializing");
+        if (!elevator.isCalibrated()) {
+            log.info("Elevator uncalibrated");
```

Comment:
I feel like this should be a warning and include something like "THE ELEVATOR WILL NOT BE ABLE TO RUN UNDER AUTOMATIC CONTROL!"